Gruntfile.js

```diff
@@ -99,6 +111,12 @@ module.exports = function (grunt) {
         "mochaTest",
         "run:testRules",
     ].concat(checkBinTest));
+    grunt.registerTask("docs", [
```

Comment:
It should be run as part of our verification step on every commit -- we want to make sure the docs TS code doesn't get broken